definitions/push-application-snapshot/push-application-snapshot.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-application-snapshot-task
+spec:
+  params:
+    - name: mappedApplicationSnapshot
+      description: |
+        JSON string representing the ApplicationSnapshot
+        For example:
```

Comment:
Too much info here. Could you take a look to #25? We are adding a README.md file with a short description. Copy the one that Johnny has there and change what's needed please. Easier for external users to see how this works.

Reply:
agreed - didn't know we could put markdown in bundle dirs, and I really like that!
Fully documented - PTAL